copylot/hardware/cameras/Basler/bCam.py

```diff
@@ -0,0 +1,67 @@
+from pypylon import pylon as py
+from pypylon import genicam
+import sys
+
+
+
+exitcode = 0
+
+
+class bCam:
```

Comment:
Let's do `BaslerCamera` for the class name and it should extend our camera Abstract Class

Reply:
I am wondering if this should be more specific ..because the usb Basler may require a different way of operating it